package.json

```diff
@@ -47,7 +47,6 @@
     "route action"
   ],
   "dependencies": {
-    "ember-getowner-polyfill": "^1.0.0",
```

Comment:
The dep is still required (for backwards compat to < ember@2.3), can you bring it back and bump to the latest version?